modules/patient-portal/src/main/frontend/src/patient-portal/Clinics.jsx

```diff
@@ -78,8 +78,8 @@ function Clinics(props) {
           inProgress: (dialogOpen && isNewClinic)
         }}
         columns={columns}
-        entryType={"Proms/ClinicMapping"}
-        customUrl={"Proms/ClinicMapping.paginate"}
+        entryType={"Patient/ClinicMapping"}
+        customUrl={"Patient/ClinicMapping.paginate"}
```

Comment:
Shouldn't this be
```suggestion
        entryType={"Survey/ClinicMapping"}
        customUrl={"Survey/ClinicMapping.paginate"}
```
?